baggage/baggage.go

```diff
@@ -289,45 +289,37 @@ func parseMember(member string) (Member, error) {
 		props      properties
 	)
 
-	parts := strings.SplitN(member, propertyDelimiter, 2)
-	switch len(parts) {
-	case 2:
+	part0, part1, found := strings.Cut(member, propertyDelimiter)
```

Comment:
How about renaming to

```suggestion
	keyValue, properties, found := strings.Cut(member, propertyDelimiter)
```

Reference: https://www.w3.org/TR/baggage/